CONTRIBUTING.md

```diff
@@ -1,53 +1,64 @@
 # Contributing
 
-We love pull requests from everyone.
-By participating in this project,
+We love pull requests from everyone.  By participating in this project,
 you agree to abide by the thoughtbot [code of conduct].
 
   [code of conduct]: https://thoughtbot.com/open-source-code-of-conduct
 
-We expect everyone to follow the code of conduct
-anywhere in thoughtbot's project codebases,
-issue trackers, chatrooms, and mailing lists.
+We expect everyone to follow the code of conduct anywhere in
+thoughtbot's project codebases, issue trackers, chatrooms, and mailing
+lists.
 
-## Developing
+## Getting Started
 
 After you have cloned this repo, run this setup script to set up your machine
 with the necessary dependencies to run and test this app:
 
     % ./bin/setup
 
-It assumes you have the following:
-
-* Postgresql
-* Elixir
-* PhantomJS
+It assumes you have a machine equipped with [asdf], [asdf-elixir], [asdf-erlang],
```

Comment:
We don't actually assume the plugins, we only assume asdf now that the script is updated, correct?

Reply:
To run `bin/setup` we do assume the plugins are installed, otherwise `asdf install` will fail.

Reply:
Cool.